karma.conf.js

```diff
@@ -68,6 +68,10 @@ module.exports = function(config) {
         pattern: "./node_modules/webidl2/lib/*.js",
         included: false,
         served: true,
+      }, {
```

Comment:
We shouldn't include the built version. That kinda defeats the purpose (making it extremely difficult to debug any problems). If there are differences between the two versions, we should fix those.